src/components/Nav/Nav/Nav.scss

```
@@ -0,0 +1,233 @@
+* {
+  box-sizing: border-box;
+}
+header {
+  position: relative;
+  width: 100%;
+  height: 110px;
+  z-index: 50;
+  left: 0;
+  top: 0;
+  // border: 1px solid black;
+  .top_menu {
+    width: 100%;
+    height: 40px;
+    //   border: 1px solid black;
+    background: #e5e5e5;
+    .inner {
+      width: 1200px;
+      margin: 0 auto;
+      .mynb {
+        display: block;
+        float: left;
+        margin-top: 14px;
+      }
```

- 항상 새로운 셀렉터를 시작하실 땐, 가독성을 위해 한칸 띄워주세요!
```suggestion
  .top_menu {
    width: 100%;
    height: 40px;
    //   border: 1px solid black;
    background: #e5e5e5;
    
    .inner {
      width: 1200px;
      margin: 0 auto;
      
      .mynb {
        display: block;
        float: left;
        margin-top: 14px;
      }
```